### DISCUSSION TITLE
[FIRMWARE REQUEST] Multiple instance of the extra button add on

### DISCUSSION THREAD

**Discussion by ZoddoDK** [OpenStickCommunity/GP2040-CE#383] [General]
Would it be possible to expand this Add On feature to configure multiple extra Buttons? I would really like to Configure more than one of these so I don't have to rewire my Buttons all the time if I want to have 2 Buttons to have the same Output. Cheers.

**Comment by bsstephan** [comment]
I feel like this has been talked about a couple times, but I am not finding an issue for it. Can you please check for one or make a new one?

**Reply by ZoddoDK** [reply]
So do I understand correctly that it should already be possible to configure more than one Extra Button under the AddOns?

**Reply by bsstephan** [reply]
No, we've talked about how it is currently limited to one, and could probably be expanded to multiple. But I don't think anyone bothered to actually file an issue for it, so you should so that we know people would like the feature.

**Comment by TheTrainGoes** [comment]
Hi @ZoddoDK ,

This is now rolled into a more robust pin mapping system.  You can have 30 `UP` pins if you like now.
